### PR TITLE
h1loss output sum, fix for issue #103

### PR DESCRIPTION
**Pull request by itskobold**
h1loss output is summed to produce a scalar, training would previously break if number of output channels > 1

https://github.com/neuraloperator/neuraloperator/issues/103

### CODE REVIEW

**Comment by kovachki** [comment]
For the loss to produce a scalar, the channel dimension must be added to the `reduce_dims` otherwise it will only reduce over the first dimension (this is the default behavior). Here is a 2d example:
```
loss = H1Loss(d=2, reduce_dims=[0,1])
x = torch.randn(5,10,128,128)
y = torch.randn(5,10,128,128)
loss(x,y)
```
This will be scalar.

**Comment by dhpitt** [comment]
Closing, @kovachki's explanation of we compute loss this way is sufficient
